src/test/java/GreetzLoginFavouritesTest.java

```diff
@@ -0,0 +1,134 @@
+    public void testFavouritesFunctionality() throws InterruptedException {
+        driver = createDriverInstance();
+        try {
+            testLoginImpl();
+
+            driver.get("https://www.greetz.nl/ballonnen/denken-aan");
+            Thread.sleep(3000);
+
+            WebElement chosenItem = driver.findElement(By.xpath("//div[text()=\"Ballon 'op fleuren'\"]/ancestor::a/following-sibling::a/div"));
+            chosenItem.click();
+            String expectedItemTitle = driver.findElement(By.xpath("//a[@href='/ballonnen/detail/ballon--op-fleuren-/1142785578']/div[2]")).getText();
+            String expectedItemPrice = driver.findElement(By.xpath("//div[text()=\"Ballon 'op fleuren'\"]/ancestor::a//span/span")).getText();
+            Thread.sleep(3000);
+
+            WebElement myGreetz = driver.findElement(By.linkText("MyGreetz"));
+            myGreetz.click();
+            Thread.sleep(3000);
+
+            WebElement favorieten = driver.findElement(By.xpath("//div[@class=\"hamburgermenu profilehamburgermenu hamburgermenu_visible\"]//span[text()='Favorieten']"));
+            favorieten.click();
+            Thread.sleep(3000);
+
+            WebElement favItem = driver.findElement(By.xpath("//*[@class='favorite-item']"));
+            favItem.click();
+            Thread.sleep(3000);
+
+            String actualItemTitle = driver.findElement(By.xpath("//*[@ng-bind=\"gift.title\"]")).getText();
+            String actualItemPrice = driver.findElement(By.xpath("//*[@class=\"price-normal\"]")).getText();
+            Assert.assertTrue(actualItemPrice.contains(expectedItemPrice), "Prices of basket item: " + actualItemPrice + " and of chosen item: " + expectedItemPrice + " are different.");
+            Assert.assertTrue(actualItemTitle.contains(expectedItemTitle), "Names of basket item: " + actualItemTitle + " and chosen item: " + expectedItemTitle + "are different.");
+        } finally {
+            driver.quit();
+        }
+    }
+
+    @Test
+    public void testCardsPricingChange() throws InterruptedException {
+        driver = createDriverInstance();
+        try {
+            testLoginImpl();
+
+            driver.get("https://www.greetz.nl/kaarten/denken-aan");
+            Thread.sleep(3000);
+
+            List<WebElement> cards = driver.findElements(By.xpath("//div[@class=\"b-card-preview__container\"]"));
+            WebElement randomChosenItem = cards.get(getRandomNumberUsingNextInt(cards.size()));
+            randomChosenItem.click();
+            Thread.sleep(3000);
+            WebElement randomChosenItemQuantity = driver.findElement(By.xpath("//input[@type='number']"));
+            int chosenQuantity = getRandomNumberUsingNextInt(10) + 1;
+            randomChosenItemQuantity.clear();
+            randomChosenItemQuantity.sendKeys("" + chosenQuantity);
+            Thread.sleep(2000);
+            String price = driver.findElement(By.xpath("//*[@class='price-card price-block']/span/span")).getText();
```

Comment:
You could simply use `price-normal` class

Reply:
Addressed in 7968d48.

Applied `xpath("//*[@class='price-normal']")`